### DISCUSSION TITLE
how use @sentry/nextjs and next-with-workbox together

### DISCUSSION THREAD

**Discussion by behnam-maqsudi** [vercel/next.js#26937] [Help]
### What version of Next.js are you using?

11.0.0

### What version of Node.js are you using?

14.17.0

### What browser are you using?

chrome

### What operating system are you using?

window 10

### How are you deploying your application?

other

### Describe the Bug

next.config.js
```
const { i18n } = require('./next-i18next.config');
const { withSentryConfig } = require('@sentry/nextjs');
const withWorkbox = require('next-with-workbox');
const withPlugins = require('next-compose-plugins');

const SentryWebpackPluginOptions = {};

const moduleExports = withWorkbox({
  workbox: {
    swSrc: 'worker.js',
  },
  i18n,
  webpack5: false,
  eslint: {
    ignoreDuringBuilds: true,
  },
});

module.exports = withSentryConfig(moduleExports, SentryWebpackPluginOptions);
```

### Expected Behavior

when i want build i have some issue like this
```
Failed to compile.

You must provide the URL of lib/mappings.wasm by calling SourceMapConsumer.initialize({ 'lib/mappings.wasm': ... }) before using SourceMapConsumer


> Build error occurred
Error: > Build failed because of webpack errors
    at C:\Users\psp\Desktop\pwa\node_modules\next\dist\build\index.js:15:924
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
    at async Span.traceAsyncFn (C:\Users\psp\Desktop\pwa\node_modules\next\dist\telemetry\trace\trace.js:6:584)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

```

### To Reproduce

a

**Comment by jasonadkison** [comment]
Having the same issue, were you able to find a solution?

FWIW I can reproduce the error without `@sentry/nextjs` - when building workbox + Webpack 4 + `productionBrowserSourceMaps: true`.

One recommended fix is to upgrade to Webpack 5 and change the `target` from `serverless` to `experimental-serverless-trace`, but for people who may not be able to upgrade to Webpack 5 I don't know that a solution currently exists.
